python-modules/python-scandir-1.5/test/test_scandir.py

```diff
@@ -133,8 +133,12 @@ def test_stat(self):
             os_stat = os.stat(os.path.join(TEST_PATH, entry.name))
             scandir_stat = entry.stat()
             self.assertEqual(os_stat.st_mode, scandir_stat.st_mode)
-            self.assertEqual(os_stat.st_mtime, scandir_stat.st_mtime)
-            self.assertEqual(os_stat.st_ctime, scandir_stat.st_ctime)
+            # The following original tests fail in Windows sometimes.
+            # Reported upstream at https://github.com/benhoyt/scandir/issues/77.
+            # self.assertEqual(os_stat.st_mtime, scandir_stat.st_mtime)
+            # self.assertEqual(os_stat.st_ctime, scandir_stat.st_ctime)
+            self.assertEqual(round(os_stat.st_mtime, 3), round(scandir_stat.st_mtime, 3))
```

Comment:
there is a dedicated assertion

https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertAlmostEqual

```
self.assertAlmostEqual(os_stat.st_mtime, scandir_stat.st_mtime, delta=1)
```

as a best practice is always better to you use dedicated assertion, as if something fails, the dedicated error can present a most helpful error message ... and the error can show the actual actual and expected values